user/store.go

```diff
@@ -129,7 +148,16 @@ func NewDB(ctx context.Context, db *sql.DB) (*DB, error) {
 				provider_id = $2 AND 
 				subject_id = $3
 		`),
-	}, p.Err
+	}
+	if p.Err != nil {
+		return nil, p.Err
+	}
+
+	store.userExist <- make(map[uuid.UUID]struct{}, 5000)
```

Comment:
would it be excessive to allocate enough capacity with some variation of `select count(*) users`?

Reply:
It wouldn't be worth the lookup/roundtrip to the DB at startup just for that. The map will auto-expand if it goes beyond this value though. 5k will only use up approx. 80KiB.

Honestly though, it might not be worth pre-allocating in the first place -- it's arbitrary, doesn't have much performance impact (one-time deal), and thus might just cause confusion/clutter. Will remove.